<launch>
    <node pkg="rms_scara" type="scara2.py" name="scara2_node" output="screen">
    </node>

    <!--node name="image_view" pkg="image_view" type="image_view" respawn="false" output="screen">
        <remap from="image" to="/image_processed" />
    </node-->
</launch>
